<ui version="4.0" >
 <class>RobotDialog</class>
 <widget class="QDialog" name="RobotDialog" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Robot Demo</string>
  </property>
  <property name="windowIcon" >
   <iconset>icon/robot.png</iconset>
  </property>
  <property name="sizeGripEnabled" >
   <bool>false</bool>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="" >
   <property name="geometry" >
    <rect>
     <x>20</x>
     <y>20</y>
     <width>661</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QLabel" name="mapView" >
      <property name="cursor" >
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="acceptDrops" >
       <bool>false</bool>
      </property>
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text" >
       <string/>
      </property>
      <property name="scaledContents" >
       <bool>true</bool>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="MapButton" >
        <property name="text" >
         <string>&amp;Map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NextButton" >
        <property name="text" >
         <string>&amp;Next</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="QuitButton" >
        <property name="text" >
         <string>&amp;Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitButton</sender>
   <signal>clicked()</signal>
   <receiver>RobotDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>491</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel" >
     <x>275</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
